isofit/radiative_transfer/radiative_transfer.py

```diff
@@ -81,6 +82,18 @@ def __init__(self, full_config: Config):
         for idx in range(len(config.radiative_transfer_engines)):
             confRT = config.radiative_transfer_engines[idx]
 
+            """Handle glint model flag from the surface model and
+            the topography model for a multipart-transmittance lut
+            Should be depreciated with #524?
```

Comment:
Yes, correct! With #524, ISOFIT will be able to obtain any surface specific information from the surface model itself without the need to set a flag in the config file. In addition, the topography model will be applied automatically as soon as the user provides a LUT with separated transmittance quantities, so no flag required here either.